core/modules/wildcard_match.cc

```diff
@@ -397,30 +407,33 @@ CommandResponse WildcardMatch::CommandGetRules(const bess::pb::EmptyArg &) {
     }
     f->set_num_bytes(field.size);
   }
+}
+
+// Sets up default gate, adds a rule for each (gate, priority) entry,
+// and then calls put_field on each (field_mask, field_match, size).
+template <typename rule_t, typename T, typename Func>
+void WildcardMatch::GetRules(T &resp, Func put_field) {
```

Comment:
can this be a const function?

Reply:
I think so (or just all moved inline when we ditch the extra templates...).